compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -343,6 +355,71 @@ internal class ComposeSceneMediator(
             }
         }
 
+    fun onScrollEvent(
+        position: DpOffset,
+        velocity: DpOffset,
```

Comment:
```suggestion
        delta: DpOffset,
```

Looking at the code, we send not a velocity (delta/deltaTime), but the actual delta (and we should send it, not velocity)

Reply:
Renamed